common/driver/gpu_async.c

```diff
@@ -329,10 +329,10 @@ void Gpu_Show(struct seq_file *s, struct DmaDevice *dev) {
    seq_printf(s, "           Write Enable : %d\n", writeEnable);
    seq_printf(s, "      Read Buffer Count : %d\n", readBuffCnt);
```

Comment:
@JJL772 While you are at it, can you update the integer type of other non-negative values as well?  E.g. readBuffCnt is also never negative as well and probably should be `%u\n`

Reply:
Done